e2e/pages/events/uploadC2DocumentsEvent.page.js

```diff
@@ -12,6 +13,19 @@ module.exports = {
     pbaNumber: '#temporaryC2Document_pbaNumber',
     clientCode: '#temporaryC2Document_clientCode',
     customerReference: '#temporaryC2Document_fileReference',
+    supportingDocuments: {
+      name: '#temporaryC2Document_supportingEvidenceBundle_0_name',
+      notes: '#temporaryC2Document_supportingEvidenceBundle_0_notes',
+      dateAndTime: {
+        day: '#temporaryC2Document_supportingEvidenceBundle_0_dateTimeReceived-day',
+        month: '#temporaryC2Document_supportingEvidenceBundle_0_dateTimeReceived-month',
+        year: '#temporaryC2Document_supportingEvidenceBundle_0_dateTimeReceived-year',
+        hour: '#temporaryC2Document_supportingEvidenceBundle_0_dateTimeReceived-hour',
+        minute: '#temporaryC2Document_supportingEvidenceBundle_0_dateTimeReceived-minute',
+        second: '#temporaryC2Document_supportingEvidenceBundle_0_dateTimeReceived-second',
+      },
+      document: '#temporaryC2Document_supportingEvidenceBundle_0_document',
+    },
```

Comment:
I think since this is a CCD collection, your e2e page object should support the ability of adding multiple. Even if you don't make use of it yourself it can potentially help others down the line.

Reply:
`supportingDocuments` can be turned into a function which takes an index, this index can be then used in the place of `0`, e.g.
```js
    supportingDocuments: (index) => ({
      name: `#temporaryC2Document_supportingEvidenceBundle_${index}_name`,
    }),

```

Reply:
I tried exactly that before you added your comment but it didn't work and reverted back to look at later. Let me try again!

Reply:
it was just a guess, but I know that we have done it else where

Reply:
See [sendCaseToGatekeeperEvent.page.js](https://github.com/hmcts/fpl-ccd-configuration/tree/master/e2e/pages/events/sendCaseToGatekeeperEvent.page.js)

Reply:
done